### DISCUSSION TITLE
Error: Include file '/home/pi/printer_data/config/./cfgs/misc-macros.cfg' does not exist

### DISCUSSION THREAD

**Discussion by Altduckthing** [bassamanator/Sovol-SV06-firmware#33] [Q&A]
I'm having trouble with this error I have no idea where to start
[klippy.log.txt](https://github.com/bassamanator/Sovol-SV06-firmware/files/11246216/klippy.log.txt)

**Comment by bassamanator** [comment]
You missed something in the instructions. Delete everything and start from the Download section again.

**Reply by Altduckthing** [reply]
Ok so I forgot to include the mainsail.cfg thing so I got it working now

**Reply by bassamanator** [reply]
Great!
